### PR TITLE
Add types for easycaptchajs package

### PR DESCRIPTION
**Pull request by HichemTab-tech**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If adding a new definition:
- [x] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [x] If this is for an npm package, match the name. If not, do not conflict with the name of an npm package.
- [x] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] Represents shape of module/library [correctly](https://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html)
- [x] `tslint.json` [should contain](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#linter-tslintjson) `{ "extends": "@definitelytyped/dtslint/dt.json" }`, and no additional rules.
- [x] `tsconfig.json` [should have](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#tsconfigjson) `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: <<url here>>
- [x] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

If removing a declaration:
- [x] If a package was never on Definitely Typed, you don't need to do anything. (If you wrote a package and provided types, you don't need to register it with us.)
- [x] Delete the package's directory.
- [x] Add it to `notNeededPackages.json`.

### CODE REVIEW

**Comment by HichemTab-tech** [comment]
when the PR will be merged ?

File: types/easycaptchajs/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+    token: string
+}
+
+export type EasyCaptchaResult = EasyCaptchaResultTarget | EasyCaptchaResultVerify | EasyCaptchaResultResponse;
+
+export type EasyCaptchaResults = EasyCaptchaResult[] | EasyCaptchaResult | null;
+
+export interface EasyCaptchaInstance {
+    (options?: EasyCaptchaSettings, ...args: string[]): EasyCaptchaResults;
+    (method: EasyCaptchaMethod, results: any[], data: any, args: string[]): EasyCaptchaResults;
```
Comment (peterblazejewicz) [inline types/easycaptchajs/index.d.ts:54]: isn't that 'data' to an an object with at least 'parentId'/'widgetId' properties? (I only point here, it should be at least `Record<string, unknown>` or similar, not just any, as it cannot be a scalar type, like boolean, number, etc

Comment (HichemTab-tech) [reply types/easycaptchajs/index.d.ts]: Thanks for pointing that to me, i fixed it, now data have a defined type (`EasyCaptchaResult`) and not an `any`.

**Review by peterblazejewicz — APPROVED** [review]
LGTM!
@HichemTab-tech thanks!

**Comment by HichemTab-tech** [comment]
Ready to merge

**Pull request merged**